data_validation/result_handlers/text.py

```diff
@@ -36,11 +37,11 @@ def print_formatted_(self, result_df):
         :param result_df
         """
         if self.format == "text":
-            print(result_df.to_string(index=False))
+            logging.info(result_df.to_string(index=False))
```

Comment:
Does the logging change the nature of this printing (newlines might either split into new logs or be less exportable)

I think printing might be more logical here, but we need to maintain the ability to export into a file

Reply:
I don't think logging would split into new logs if it has \n in it. 
https://stackoverflow.com/questions/22934616/multi-line-logging-in-python.
Do let me know if this makes sense to you

Reply:
It something I've seen in stackdriver specifically.

In any case, I'm more worried that you are changing the actual output here since this option is requesting a specific type to be output.  CSV and json data would no longer be valid, so anyone piping these out to files would start seeing differences.

Reply:
Ahh makes sense!